calculator_bronze.rb

```diff
@@ -0,0 +1,42 @@
+puts "Hello! Welcome to my calculator!\nWhat kind of math operation would you like to use?"
+
+#what operation do they want to use? Get user input
+
+operation = gets.chomp
+
+#make an if/elsif statement for each operation. Add, subtract, multiply, divide
+if (operation.downcase == "addition") || (operation.downcase == "add") || (operation == "+")
```

Comment:
Hi there! I'm from cohort[1] :cd: 

This is a good use of if/elsif/else statements along with those ||'s and =='s. Maybe consider using case (switch) statements (http://www.codecademy.com/glossary/ruby/switch-statement), but these if/elsif/else statements work just fine.